### DISCUSSION TITLE
java.lang.NoClassDefFoundError: GraphQLResponseProjection

### DISCUSSION THREAD

**Discussion by deathsurgeon1** [kobylynskyi/graphql-java-codegen#982] [Q&A]
While integration I am getting the error mentioned in Title. Any suggestions to resolve this.

**Comment by kobylynskyi** [comment]
Can you show you complete pom.xml/build.grade?

**Reply by deathsurgeon1** [reply]
pom is huge . I am not sure if i would be able to share

**Reply by kobylynskyi** [reply]
Make sure you have dependency included (not just plugin)

**Reply by deathsurgeon1** [reply]
`<dependency>
            <groupId>io.github.kobylynskyi</groupId>
            <artifactId>graphql-java-codegen</artifactId>
            
            <version>4.1.2</version>
        </dependency>`
Dependency is included in pom

**Reply by kobylynskyi** [reply]
What is the version of the plugin you are using?
Try making upgrading to the latest version of both plugin and library (currently [5.4.1](https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v5.4.1))
